.github/workflows/build.yml

```diff
@@ -60,6 +60,9 @@ jobs:
           distribution: 'zulu'
           java-version: 14
 
+      - name: Setup XCode
+        run: sudo xcode-select -switch "/Applications/Xcode_12.app"
```

Comment:
There was error in multiplatform about xcode not found and stuff so I added it and now it's green.

Reply:
That makes sense. Are those paths (like `Xcode_12.app`) documented from GitHub?

Also, nit: It's `Xcode` with a lower-case "C"